src/ParseError.js

```diff
@@ -433,6 +433,15 @@ ParseError.EMAIL_MISSING = 204;
  */
 ParseError.EMAIL_NOT_FOUND = 205;
 
+/**
+ * Error code indicating that the email has already been verified and can not be verified again.
+ *
+ * @property {number} EMAIL_ALREADYـVERIFIED
+ * @static
+ */
+ParseError.EMAIL_ALREADYـVERIFIED = 212;
```

Comment:
Please check the underscore characters, the 2nd one is an incorrect char, in comment and code.

Reply:
oh, sorry.
I replaced 2nd one with correct underscore characters.